### ISSUE TITLE
production.clinicalgenome.org should not show demo/test notice

### ISSUE THREAD

**Issue by kilodalton** [ClinGen/clincoded#467]
`*.production.clinicalgenome.org` instances should not show usual demo notice and also should not show any test interface messages.

Perhaps an alternate message that refers to production or no message/bar at all.

**Comment by mrmin123** [comment]
To start things off, I made it so that on any `*production.clinicalgenome.org` URLs the test-interface background is off, and the red notice bar is replaced with this yellow one:

![image](https://cloud.githubusercontent.com/assets/4326866/11132557/6630f766-8945-11e5-98c3-734c9c877d8c.png)

The 'main site' links to `http://curation.clinicalgenome.org/'.

**Comment by mrmin123** [comment]
v2:

![image](https://cloud.githubusercontent.com/assets/4326866/11133779/ed240e8c-894c-11e5-819b-734a9a7917ee.png)

**Comment by selinad** [comment]
@mrmin123 - the light yellow is for the alternate production URL (for example) that is the same as curation.clinicalgenome.org and the dark yellow is for the version that only we can access, correct?

I would put the dark yellow for the alternate production URL - it should be very visible, especially since people can log in.  The light yellow could go on the other (where people can't log in, but it will serve as a reminder to us) - could also be a darker color different than red or yellow (bright green, like that found in header?).

I also have thoughts on the messages:
# For production url - I'd repeat the message in your warning bar rather than saying access is limited to ClinGen curators - ClinGen curators might log in (and can).  Then add the sentence "To access publicly available information, please visit the ClinGen portal."

# For our internal URL - "Access to this site is restricted.  If you are a ClinGen curator, please go to the following URL to either request access or. if you are already registered, to curate information: curation.clinicalgenome.org.  To access publicly available information, please visit the ClinGen portal.

My 2 cents.

**Comment by mrmin123** [comment]
@selinad Right now the big yellow bar is for any site that's served via a `production.clinicalgenome.org` address. Not sure what this 'internal' URL is/would be...

**Comment by selinad** [comment]
Which big yellow bar?  You have 2, so I am confused :confused:

**Comment by selinad** [comment]
Maybe both images are for the production - sorry.  If that's the case, I like the darker yellow one and change the text below Not Available to something like I suggested above.

I was thinking you were also doing a mockup for the site that only our group can access (as there was no accompanying text :wink: )

**Comment by mrmin123** [comment]
@selinad Ah, no, sorry, should have been more clear. the second 'v2' image was to denote that I had replaced the previous screenshot's implementation. Basically, disregard the first image other than for reference.

Actually, let me add a new screenshot. The 'Not Available' isn't specific to a URL... it was just the page I was on when I took the screenshot.

**Comment by mrmin123** [comment]
OK. This is the current implementation I have for the notice bar for all `production.clinicalgenome.org` sites:

![image](https://cloud.githubusercontent.com/assets/4326866/11134766/340db4c2-8954-11e5-928d-a8457a95a620.png)

**Comment by kilodalton** [comment]
@selinad It's only an edge case that someone besides us would ever be on a "production.clingen" URL. We discussed it and thought it should it should be simple, bold, and not interfere with the site. If it is too wordy people will not read the warning message.

**Comment by selinad** [comment]
Thanks @mrmin123 - sorry, makes sense!  I completely missed the v2 (maybe it was the bright yellow, which is good!).

Should we say something on the bar like:

Do not use this URL for entering data. Please use curation.clinicalgenome.org.

**Comment by selinad** [comment]
@kilodalton - the text is for under the Not Available.  Agreed - I like short  text on the banner.

**Comment by selinad** [comment]
And, apologies, as I should have been more clear! :wink:

**Comment by mrmin123** [comment]
@selinad 
![image](https://cloud.githubusercontent.com/assets/4326866/11135128/06de806e-8957-11e5-935f-22cacc9e27e2.png)

**Comment by selinad** [comment]
I like it!  What do others think?  I think it's good to explicitly say not to enter data here (and there's no reason to explain why).

Would recommend changing text for non-logged in page as well (under Not Available).

**Comment by mrmin123** [comment]
@selinad The 'Not Available' is for all HTTPForbidden pages across the site: when a user, logged in or not, attempts to access a page they're not supposed to have access to. This theoretically only happens when a logged out user attempts to view a page they need to be logged in to see, but again, it's not specific to the production site/URL.

Do you still want the line '"To access publicly available information, please visit the ClinGen portal."' to be added site-wide?

**Comment by selinad** [comment]
Hi @mrmin123 - I think you are saying you cannot customize this text for the URL and production vs. non-production?

**Comment by mrmin123** [comment]
@selinad It's customizable, I guess I just don't really see the use in it? It's a page that's so rarely seen, unless you're constantly logging in/out like us, anyway. Unless with the possibility of someone being linked to a curation item under the `production.clinicalgenome.org` URL...

**Comment by selinad** [comment]
what if someone happens to land on the URL, which is the small probability case we are using this for - if they are not logged in they will see it - if they are logged in, now I can't remember - they will see the dashboard?  It seems very unlikely that a logged in person would get to this page....

**Comment by mrmin123** [comment]
It's mainly dependent on *where* they land. If they land on the root page, they'll see the generic 'you have to log in' message. If they're somehow linked to something like a GDM on that URL, then they'll see the 'Not available' message. If it's literally the first time they're accessing the page, they won't be logged in since it's a different domain than the publicly available site.

**Comment by kilodalton** [comment]
Another great release (R2). Nice job and thanks for your hard work.
